### PR TITLE
Add Spanish Event Flags Translation / Update Spanish Translation

### PR DESCRIPTION
**Pull request by XxPhoenix1996xX**

### CODE REVIEW

**Comment by egzn** [comment]
I would change all "Single(s)" to Indiv. or Individual(es)

File: PKHeX.Core/Resources/text/script/flags_xy_es.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+0287	Triples Statuette
+0288	Rotación Statuette
+0289	Multi Statuette
+0290	Super Singles Desbloqueado
+0291	Super Dobles Desbloqueado
+0292	Super Triples Desbloqueado
+0293	Super Rotación Desbloqueado
+0294	Super Multi Desbloqueado
+0675	Racha de 50: Derrota Dama del Combate
+2546	Pokédex Obtenido
```
Comment (egzn) [inline PKHeX.Core/Resources/text/script/flags_xy_es.txt]: PokéDex is considered a female noun in Spanish

File: PKHeX.Core/Resources/text/script/flags_usum_es.txt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+4425	(UM) Yveltal Capturado
+1627	Solgaleo/Lunala Capturado
+1552	Necrozma Capturado
+4370	(UM) Stakataka en Bosque de Poni
+4371	(US) Blacephalon en Bosque de Poni
+4545	Debloqueado Lanturn 360
+4546	Debloqueado Primarina Twist
+4547	Debloqueado Starmie 720
+4548	Debloqueado Over-the-Gyarados
+4490	Usar Control de Pad Circular para el Ultra Warp Ride
```
Comment (egzn) [inline PKHeX.Core/Resources/text/script/flags_usum_es.txt]: Control de Pad Circular => Joystick
Ultra Warp Ride => Ultraviaje dimensional

**Comment by XxPhoenix1996xX** [comment]
Thanks @egzn , typos have been fixed and translation enhanced.

File: PKHeX.Core/Resources/text/script/const_usum_es.txt [hunk 3]
```diff
@@ -4,4 +4,4 @@
 434	Tapu Bulu	01:Luchable,02:Derrotado,03:Capturado
 404	Tapu Fini	03:Luchable,04:Derrotado,05:Capturado
 755	Zygarde	01:Luchable,02:Derrotado,05:Capturado
-867	Regalos de Tamaño-Tomtem 00:Ninguno,01:Uno Recibido,02:Dos Recibido,03:Tres Recibido,04:Cuatro Recibido,05:Cinco Recibido,06:Todos Recibidos
+867	Regalos de Tamaño-Tótem 00:Ninguno,01:Uno Recibido,02:Dos Recibido,03:Tres Recibido,04:Cuatro Recibido,05:Cinco Recibido,06:Todos Recibidos
```
Comment (egzn) [inline PKHeX.Core/Resources/text/script/const_usum_es.txt]: Totem Pokémon in Spanish are Pokémon Dominante, I would put "Dominantes por Regalo" or better, "Dominantes conseguidos con Dominsignias"; also I would swap "[Number] Recibido" to "Conseguido [Number]"

File: PKHeX.Core/Resources/text/script/flags_frlg_es.txt [hunk 4]
```diff
@@ -97,8 +97,8 @@
 0x2DC	Recibido Zafiro
 0x2FD	Recibida Piedra Eterna (Ayudante de Oak, Centro Pokemon del Túnel Roca)
 0x256	Recibido Buscaobjetos (Ayudante de Oak, Ruta 11)
-0x256	Recibido Exp. Share (Ayudante de Oak, Ruta 15)
-0x2FD	Recibido Amulet Coin (Ayudante de Oak, Ruta 16)
+0x256	Recibido Comp. Exp. (Ayudante de Oak, Ruta 15)
```
Comment (egzn) [inline PKHeX.Core/Resources/text/script/flags_frlg_es.txt]: Exp. Share in Spanish is "Repartir Exp." (though Rep. Exp. works too, I guess!)

**Comment by XxPhoenix1996xX** [comment]
@egzn thanks! Translation updated (again)

**Comment by kwsch** [comment]
Thank you both!

**Pull request merged**